src/main/java/utils/CalculationUtils.java

```diff
@@ -0,0 +1,37 @@
+package utils;
+
+import data.UserTableRow;
+import io.cucumber.datatable.DataTable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class CalculationUtils {
+
+    private CalculationUtils() {
+    }
+
+    public static String summ(String valueOne, String valueTwo) {
+        return String.valueOf(Integer.parseInt(valueOne) + Integer.parseInt(valueTwo));
+    }
+
+    public static List<UserTableRow> convertToUserTableRowList(List<String> numbers, List<String> usernames, List<String> description) {
+        List<UserTableRow> userTable = new ArrayList<>();
+        for (int i = 0; i < numbers.size(); i++) {
+            userTable.add(new UserTableRow(numbers.get(i), usernames.get(i), description.get(i)));
+        }
+        return userTable;
+    }
+
+    public static List<UserTableRow> convertUserDataToList(DataTable dataTable) {
```

Comment:
Why do you pass `DataTable` as parameter here?

Reply:
remade all that stuff with the help of POJO from params